javascript-data-model/order-history.js

```diff
@@ -0,0 +1,69 @@
+var orderHistory = [
+  {
+    placed: 'August 4, 2020',
+    total: '$34.00',
+    to: 'JS Masher',
+    order: '114-3941689-8772232',
+    details: '',
+    invoice: '',
+    delivered: 'Aug 8, 2020',
+    image: '',
+    title: 'Javascript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    return: 'Return window closed on Sep 7, 2020',
+    price: '31.55',
+    again: '',
+    view: '',
+    review: ''
+  },
```

Comment:
This object is meant to represent an "order", but you also have data for the items ordered mixed in here. I think you would benefit from a property named `items` that holds data for all the items being ordered.

Reply:
So would the item property include everything such as the image, title, author, return, and the price?